index.html

```diff
@@ -245,6 +247,28 @@ <h3 class="center">You can <a id="donate_link" href="#donate">donate</a> to help
     </div>
 </div>
 
+<!--FAQ-->
+<div class="section scrollspy" id="FAQ">
+<div class="container">
+<h2 class="center">F.A.Q</h2>
+  <ul class="collapsible" data-collapsible="accordion">
+    <li>
+      <div class="collapsible-header">X-ray</div>
+      <div class="collapsible-body"><span>.xray add [ID/diamond_ore] // Ores dark? Fixed in 4.20</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header">Optifine</div>
+      <div class="collapsible-body"><a href="https://www.youtube.com/watch?v=o1LHq6L0ibk">https://www.youtube.com/watch?v=o1LHq6L0ibk</a><span></span></div>
+    </li>
+    <li>
+      <div class="collapsible-header">There is a bug! (╯°□°）╯︵ ┻━┻</div>
+      <div class="collapsible-body"><span><a href="https://github.com/ImpactDevelopment/ImpactClient/issues">https://github.com/ImpactDevelopment/ImpactClient/issues</a></span></div>
+    </li>
```

Comment:
The link text doesn't have to be the URL (only the `href` does). This and the youtube link would probably be better with some descriptive title instead of a URL.